### DISCUSSION TITLE
Немецкая озвучка в формате ogg

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/gothic2_steam_fix#32] [General]
Привет. А можно пережать немецкую озвучку в ogg?

https://yadi.sk/d/ygf_KGsaUh9ckw

**Comment by Kor-Angar** [comment]
> а почему русская озвучка Акеллы так мало весит? :-)

> формат ogg

так это из-за этого у меня игра вылетала два раза в час, когда я диалоги прокликивал? :-) на немецкой озвучке ни разу не вылетела ;-)

**Comment by dosinabox** [comment]
> так это из-за этого у меня игра вылетала два раза в час, когда я диалоги прокликивал? :-)

крайне маловероятно
